remove bundled getopt and use libs from glibc/musl instead
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,6 +1,6 @@
 bin_PROGRAMS = mboxgrep
 mboxgrep_SOURCES = info.c info.h maildir.c maildir.h main.c mbox.c mbox.h \
 									 mboxgrep.h md5.c md5.h message.h misc.c misc.h mh.c mh.h \
-									 getopt.c getopt1.c getopt.h re.c re.h scan.c scan.h wrap.c wrap.h
+									 re.c re.h scan.c scan.h wrap.c wrap.h
 
 mboxgrep_LDADD = $(LIBOBJS)
